.github/workflows/contracts-random-beacon.yml

```diff
@@ -334,9 +349,24 @@ jobs:
       
       - name: Deploy contracts
         env:
-          CHAIN_API_URL: ${{ secrets.GOERLI_ETH_HOSTNAME_HTTP }}
-          ACCOUNTS_PRIVATE_KEYS: ${{ secrets.DAPP_DEV_GOERLI_ETH_CONTRACT_OWNER_PRIVATE_KEY }}
-          ETHERSCAN_API_KEY: ${{ secrets.ETHERSCAN_API_KEY }}
+          # Use fake ternary expressions to decide which credentials to use,
+          # depending on chosen environment. Note: if `GOERLI...` credentials
+          # are empty, the expressions will be evaluated to the `SEPOLIA...`
+          # ones.
+          CHAIN_API_URL: |
+            ${{ inputs.github.event.inputs.environment == 'goerli'
+              && secrets.GOERLI_ETH_HOSTNAME_HTTP
+              || secrets.SEPOLIA_ETH_HOSTNAME_HTTP }}
+          # TODO: Shouldn't we use `ACCOUNTS_PRIVATE_KEYS` here instead of the
+          # two below envs?
```

Comment:
Could make sense as the deployer is usually the first account. Also, even if we don't use `ACCOUNTS_PRIVATE_KEYS` having two separate envs for Goerli and Sepolia feels a bit redundant as those will likely be the same account.

Reply:
Decided to use it, see https://github.com/keep-network/keep-core/pull/3706/commits/31367734ef7257213faa88b1aa53b372de27ca20.